### PR TITLE
Light wallet i phone sideloading guide

### PR DESCRIPTION
**Pull request by ohcee**
Light-wallet iPhone sideloading guide, light-wallet photos, and updated CSS.

### CODE REVIEW

File: Veil-light-wallet-iPhone-sideload.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    <h2>Downloading the Veil Light Wallet:</h2>
+        <ol>
+            <li>Visit GitHub to obtain the latest version of the light wallet <a href="https://github.com/steel97/veil_wallet/releases" target="_blank" rel="noopener noreferrer">here</a>.</li>
+            <ul>For iOS it Should look like this:</ul>
+            <figure>
+                <img src="light-wallet-pictures/veil_wallet_ipa.png" alt="veil_wallet_ios_ipa_photo">
+            </figure>   
+            <li>Download the IPA file to your computer. <br>
+                In this demonstration I'm unpacking and moving the IPA file to my iCloud on my mac.</li>
```
Comment (steel97) [inline Veil-light-wallet-iPhone-sideload.html]: Release builds aren't packed to zip like in actions tab, so it's better to use just "moving the IPA file" without "unpacking"

File: Veil-light-wallet-iPhone-sideload.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        </ol>
+
+    <h2>Launching Veil Light Wallet:</h2>
+        <ol>
+            <li>Press the Veil icon to open the Veil Light Wallet on your iPhone. It should be available on you homescreen.</li>
+            <li>Configure any initial settings, such as language.</li>
+            <li>Ensure you record and securely store your seed phrase generated during the setup. You can also enter a seed Phrase from a previous Veil wallet.</li>   
+            <li>After entering your wallet phrase correctly let your wallet load.</li> 
+            <li>The wallet should be ready to use in a few seconds up to a 1/2 hour.</li>
+            <li>Note that since Veil is private by default, this light wallet uses RingCT, and basecoin will not be usable, visible, or available with this light wallet.</li>
```
Comment (steel97) [inline Veil-light-wallet-iPhone-sideload.html]: basecoin and CT, only RingCT available for now (in future there probably will be option to automatically convert CT to RingCT, but now it's unavailable)

**Review by steel97 — COMMENTED** [review]
Looks good for me, just some minor notes, thanks for this guide!

File: Veil-light-wallet-iPhone-sideload.html (hunk 2 above)
Comment (ohcee) [reply Veil-light-wallet-iPhone-sideload.html]: Updated.

File: Veil-light-wallet-iPhone-sideload.html (hunk 1 above)
Comment (ohcee) [reply Veil-light-wallet-iPhone-sideload.html]: Fixed it. Thanks.

**Review by steel97 — APPROVED** [review]

**Pull request merged**